gulpfile.mjs

```diff
@@ -1682,13 +1682,13 @@ gulp.task(
   )
 );
 
-function compressPublish(targetName, dir) {
+function compressPublish({ sourceDirectory, targetFile, modificationDate }) {
   return gulp
-    .src(dir + "**", { encoding: false })
-    .pipe(zip(targetName))
+    .src(`${sourceDirectory}**`, { encoding: false })
+    .pipe(zip(targetFile, { modifiedTime: modificationDate }))
```

Comment:
Nit: If the above is implemented.
```suggestion
    .pipe(zip(targetFile, { modifiedTime }))
```

Reply:
Good idea; I've made the change in the new commit.